source/_patterns/01-elements/headline/headline.scss

```diff
@@ -1,79 +1,79 @@
 @use "sass:math";
 @import "headline.variables";
 
-.elm-headline {
-	margin-bottom: to-rem($pxValue: 16);
+// .elm-headline {
+// 	margin-bottom: to-rem($pxValue: 16);
 
-	// * TODO: These declarations make sense for the moment, but should get aligned with overall definitions of spacing around elements
-	margin-top: to-rem($pxValue: 24);
-}
+// 	// * TODO: These declarations make sense for the moment, but should get aligned with overall definitions of spacing around elements
+// 	margin-top: to-rem($pxValue: 24);
+// }
 
-h1,
-h2,
-h3 {
-	&.elm-headline {
-		font-family: $db-font-family-headline;
-		font-weight: 900;
-	}
-}
+// h1,
+// h2,
+// h3 {
+// 	&.elm-headline {
+// 		font-family: $db-font-family-headline;
+// 		font-weight: 900;
+// 	}
+// }
```

Comment:
As we don't need these and the other declarations below any more, let's just remove it on total. Due to the versioning system we could still understand any changes, but would make our changes more clearer. In case that you're just "commenting out" this code piece and would need some rework or have questions, please leave a `TODO` comment